objcgen/objcgenerator-helpers.cs

```diff
@@ -12,6 +12,29 @@ public partial class ObjCGenerator {
 
 		// get a name that is safe to use from ObjC code
 
+		public static Dictionary<string, string> ObjCTypeToArgument = new Dictionary<string, string> {
```

Comment:
Part of me hates to add _more_ code to any of the objcgenerator* files.

@spouliot @timrisi - Should we hoist all of this "name geneartion code to the name generator?